torch/distributions/__init__.py

```diff
@@ -28,6 +28,21 @@
     next_state, reward = env.step(action)
     loss = -m.log_prob(action) * reward
     loss.backward()
+
+Another way to implement these stochastic/policy gradients would be to use the
+reparameterization trick from :meth:`~torch.distributions.Distribution.rsample`
+method, where the parameterized random variable can be defined as a parameterized
+deterministic function of a parameter-free random variable. The reparameterized sample
+is required to be differentiable. The code for implementing the pathwise estimation would
+be as follows::
+
+    probs = policy_network(state)
+    m = Normal(**probs)
+    # any distribution with has_rsample == True could work based on the application
```

Comment:
nit: I'd spell this `.has_rsample == True` where the dot emphasizes that `.has_rsample` is an attribute.